<?xml version="1.0" encoding="UTF-8"?>
<testsuites name="atoum testsuite">
  <testsuite name="SubscribersList" package="Vendor\Calendar\test\units" tests="7" failures="0" errors="0" skipped="0" time="0.069271326065063">
    <testcase name="testEmptyInit" time="0.0029160976409912" classname="Vendor\Calendar\test\units\SubscribersList" assertions="0"/>
    <testcase name="testNoSubChoice" time="0.0016069412231445" classname="Vendor\Calendar\test\units\SubscribersList" assertions="3"/>
    <testcase name="testPopulatedChoice" time="0.029620885848999" classname="Vendor\Calendar\test\units\SubscribersList" assertions="1"/>
    <testcase name="testMultiplePersonChoice" time="0.0069420337677002" classname="Vendor\Calendar\test\units\SubscribersList" assertions="2"/>
    <testcase name="testAddSubscribers" time="0.0073111057281494" classname="Vendor\Calendar\test\units\SubscribersList" assertions="4"/>
    <testcase name="testAddMultipleSubs" time="0.014036178588867" classname="Vendor\Calendar\test\units\SubscribersList" assertions="2"/>
    <testcase name="testWithUndisponiblePersons" time="0.0068380832672119" classname="Vendor\Calendar\test\units\SubscribersList" assertions="2"/>
  </testsuite>
  <testsuite name="DAO" package="Vendor\DAO\tests\units" tests="6" failures="0" errors="1" skipped="0" time="0.98824310302734">
    <testcase name="testRequestConnexion" time="0.022669076919556" classname="Vendor\DAO\tests\units\DAO" assertions="0"/>
    <testcase name="testCreationTables" time="0.25018811225891" classname="Vendor\DAO\tests\units\DAO" assertions="2"/>
    <testcase name="testAddPersonne" time="0.25283408164978" classname="Vendor\DAO\tests\units\DAO" assertions="4"/>
    <testcase name="testGetPersonne" time="0.21305584907532" classname="Vendor\DAO\tests\units\DAO" assertions="2"/>
    <testcase name="testUserExistant" time="0.24949598312378" classname="Vendor\DAO\tests\units\DAO" assertions="4"/>
    <testcase name="testUpdatePersonne" time="0" classname="Vendor\DAO\tests\units\DAO" assertions="3">
      <error type="Exception"><![CDATA[PDOException: SQLSTATE[42000]: Syntax error or access violation: 1064 You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near ':lastname, 
                          `prenom` = :firstname,
                 ' at line 2 in D:\m2test2\src\main\Vendor\DAO\DAO.php:204
Stack trace:
#0 D:\m2test2\src\main\Vendor\DAO\DAO.php(204): PDO->exec('UPDATE personne...')
#1 D:\m2test2\src\test\Vendor\DAO\TestDao.php(122): Vendor\DAO\DAO->updatePersonne(Object(mock\Vendor\Models\Personne), 'personneTest')
#2 phar://D:/m2test2/applis/atoum/atoum.phar/1/classes/test.php(1363): Vendor\DAO\tests\units\DAO->testUpdatePersonne()
#3 -(1): mageekguy\atoum\test->runTestMethod('testUpdatePerso...')
#4 {main}]]></error>
    </testcase>
  </testsuite>
  <testsuite name="Auth" package="Vendor\Gestion\test\units" tests="6" failures="0" errors="0" skipped="0" time="0.072109222412109">
    <testcase name="testCreateSession" time="0.0042471885681152" classname="Vendor\Gestion\test\units\Auth" assertions="0"/>
    <testcase name="testIsLogged" time="0.018123865127563" classname="Vendor\Gestion\test\units\Auth" assertions="4"/>
    <testcase name="testLogout" time="0.0089569091796875" classname="Vendor\Gestion\test\units\Auth" assertions="1"/>
    <testcase name="testConnectionOK" time="0.012647151947021" classname="Vendor\Gestion\test\units\Auth" assertions="1"/>
    <testcase name="testConectionKO" time="0.013813972473145" classname="Vendor\Gestion\test\units\Auth" assertions="1"/>
    <testcase name="testConnectionInjection" time="0.014320135116577" classname="Vendor\Gestion\test\units\Auth" assertions="1"/>
  </testsuite>
  <testsuite name="Registration" package="Vendor\Gestion\test\units" tests="2" failures="0" errors="0" skipped="0" time="0.012164831161499">
    <testcase name="testRegistration" time="1.6927719116211E-5" classname="Vendor\Gestion\test\units\Registration" assertions="0"/>
    <testcase name="testUsernameNotUsed" time="0.012147903442383" classname="Vendor\Gestion\test\units\Registration" assertions="0"/>
  </testsuite>
  <testsuite name="Mail" package="Vendor\Mailing\tests\units" tests="4" failures="0" errors="0" skipped="0" time="0.0095212459564209">
    <testcase name="testConstruct" time="0.0031161308288574" classname="Vendor\Mailing\tests\units\Mail" assertions="0"/>
    <testcase name="testGetRecipients" time="0.0022940635681152" classname="Vendor\Mailing\tests\units\Mail" assertions="6"/>
    <testcase name="testGetSetSubject" time="0.0021381378173828" classname="Vendor\Mailing\tests\units\Mail" assertions="2"/>
    <testcase name="testGetSetBody" time="0.0019729137420654" classname="Vendor\Mailing\tests\units\Mail" assertions="2"/>
  </testsuite>
  <testsuite name="MailSender" package="Vendor\Mailing\tests\units" tests="2" failures="0" errors="0" skipped="0" time="1.6630289554596">
    <testcase name="testBadSender" time="1.1952431201935" classname="Vendor\Mailing\tests\units\MailSender" assertions="0"/>
    <testcase name="testBadCredentials" time="0.46778583526611" classname="Vendor\Mailing\tests\units\MailSender" assertions="2"/>
  </testsuite>
</testsuites>
